EPRN.Waste.API/Services/JourneyService.cs

```diff
@@ -126,14 +158,53 @@ public async Task SaveReprocessorExport(int journeyId, int siteId)
 
         public async Task<WasteSubTypeSelectionDto> GetWasteSubTypeSelection(int journeyId)
         {
-            return await _wasteRepository.GetWasteSubTypeSelection(journeyId);
+            var journey = await _wasteRepository.GetWasteSubTypeSelection(journeyId);
+            if (journey == null)
+            {
+                throw new Exception(nameof(journey));
+            }
+
+            if (!journey.WasteSubTypeId.HasValue)
+            {
+                throw new Exception(nameof(journey));
+            }
+
+            var wasteSubTypeSelection = new WasteSubTypeSelectionDto
+            {
+                WasteSubTypeId = journey.WasteSubTypeId.Value,
+                Adjustment = journey.Adjustment
+            };
+
+            return wasteSubTypeSelection;
         }
 
         public async Task<string> GetWasteRecordNote(int journeyId)
         {
             return await _wasteRepository.GetWasteNote(journeyId);
         }
 
+        public async Task<JourneyAnswersDto> GetJourneyAnswers(int journeyId)
+        {
+            var journey = await _wasteRepository.GetWasteJourneyById_FullModel(journeyId);
+            if (journey == null)
+                throw new Exception(nameof(journey));
+
+            var journeyAnswersDto = new JourneyAnswersDto();
+
+            journeyAnswersDto.JourneyId = journeyId;
+            journeyAnswersDto.WhatDoneWithWaste = journey.DoneWaste == null ? string.Empty : journey.DoneWaste.Value.ToString();
+            journeyAnswersDto.BaledWithWire = journey.BaledWithWire == null ? string.Empty : (journey.BaledWithWire.Value == true ? "Yes" : "No");
+            journeyAnswersDto.Month = journey.Month == null ? string.Empty : CultureInfo.CurrentCulture.DateTimeFormat.GetMonthName(journey.Month.Value);
+            journeyAnswersDto.WasteType = journey.WasteType == null ? string.Empty : journey.WasteType.Name.ToString();
+            journeyAnswersDto.Tonnes = journey.Tonnes == null ? string.Empty : journey.Tonnes.Value.ToString();
+            journeyAnswersDto.TonnageAdjusted = journey.Adjustment == null ? string.Empty : journey.Adjustment.Value.ToString();
+            journeyAnswersDto.Note = journey.Note;
+            journeyAnswersDto.WasteSubType = journey.WasteSubType == null ? string.Empty : journey.WasteSubType.Name.ToString();
+            journeyAnswersDto.Completed = journey.Completed == null ? false : journey.Completed.Value;
```

Comment:
Again return the dto from the repository so the service layer has no access to the db model